training-react/tic-tac-toe/src/redux/game/actions.js

```diff
@@ -0,0 +1,28 @@
+import { actionsTypes, calculateWinner } from '@consts/game';
+
+const jumpToStep = step => ({
```

Comment:
Move all the functions that you export to a variable called `actionCreators` and export it